mock-postgres-dataprovider-plugin/src/main/java/io/mosip/certify/mockpostgresdataprovider/integration/repository/MockDataRepositoryImpl.java

```diff
@@ -0,0 +1,20 @@
+package io.mosip.certify.mockpostgresdataprovider.integration.repository;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import jakarta.persistence.Query;
+import org.springframework.stereotype.Repository;
+
+
+@Repository(value = "mockDataRepository")
+public class MockDataRepositoryImpl implements MockDataRepository {
```

Comment:
Name to be changed

Reply:
Changed the name of the above repo to DataProviderRepo